### ISSUE TITLE
Whisper Model

### ISSUE THREAD

**Issue by flozi00** [huggingface/text-generation-inference#985]
### Model description

Whisper is the actual most used ASR Model on the hub.
Since Idefics as multimodal model joined the tgi server I wondered if whisper could be interesting too.

Is this out of scope or would it be interesting ?

### Open source status

- [X] The model implementation is available
- [X] The model weights are available

**Comment by Narsil** [comment]
Fully out of scope.

ASR solutions are indeed interesting, and may warrant their own docker solution.
Whisper by itself to be "good" requires a lot of fiddling and custom code in order to provide "good" results (there's a lot of things trying to prevent it from hallucinating, like dynamic temperature, custom logits decoder, no batching on long files and instead serialized decoding etc...).

IMHO wav2vec2 is still highly competitive in the ASR market just because the runtime costs are basically nil compared to whisper, and it's not a decoder model (single forward pass)

Idefics was added because we needed an efficient running solution short term and it was the easiest place to add it. But it's still far from perfect nor optimized all the way. If multimodal starts to pick up a lot, we'll definitely revisit the decision and depending on usage patterns and models we see coming up we'll consider our options.

We'd rather have a few different containerized solutions which are highly efficient at what they do, rather than forcing everything into TGI and be stuck for adding features because of the already pretty large feature set which all have to work together.

**Comment by flozi00** [comment]
Sounds reasonable
Will have an eye on huggingface org to see when tgi brothers reach github ;)
